### PR TITLE
Export API Support

### PR DESCRIPTION
**Pull request by hamishcampbell**
Still WIP but basics work.

Note we should squash & merge. Don't merge as is.

### CODE REVIEW

File: koordinates/exports.py [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+    Accessor for querying and creating Exports.
+
+    Access via the ``exports`` property of a :py:class:`koordinates.client.Client` instance.
+    """
+
+    _URL_KEY = 'EXPORT'
+    _options_cache = None
+
+    @property
+    def croplayers(self):
```
Comment (rcoup) [inline koordinates/exports.py]: for consistent snake-casing I think we should do `crop_layers` (and `crop_features`)... thoughts?

Comment (craigds) [inline koordinates/exports.py]: Makes the 'crop' bit sound like a verb though, as if it's specifying 'which layers should I crop' as opposed to 'which layers should I use to crop my export'. Also it's not snake-cased in the API URL (and would look weird if it was) so I'd tend toward keeping it the same, ie `croplayers`

**Comment by rcoup** [comment]
Latest changes are on this branch.

**Comment by hamishcampbell** [comment]
Re-opening to base off `export-api`

**Pull request closed without merging**